ListComprehension/README.md

```diff
@@ -0,0 +1,305 @@
+# List Comprehension with Syntax Extensions
+
+For those of you who are addicted to the really cool
+[list comprehension features of Python](https://docs.python.org/3/tutorial/datastructures.html),
+you might be looking for something similar in Lean.
+
+First you need to understand that the Lean `List` is a `Functor`.
```

Comment:
That's a somewhat scary sentence considering that many people are familiar with `map` without ever having heard of functors.

Reply:
I've softened the language a bit, I still want to mention functors, but I removed the "need to understand" part.